tensorflow/lite/testing/op_tests/tile.py

```diff
@@ -27,21 +27,21 @@ def make_tile_tests(options):
           "input_dtype": [tf.float32, tf.int32, tf.bool, tf.string],
           "input_shape": [[3, 2, 1], [2, 2, 2]],
           "multiplier_dtype": [tf.int32, tf.int64],
-          "multiplier_shape": [[3]]
-      },
+          "multiplier_shape": [[3]],
+          "fully_quantize": [False],
       {
           "input_dtype": [tf.float32, tf.int32],
           "input_shape": [[]],
           "multiplier_dtype": [tf.int32, tf.int64],
-          "multiplier_shape": [[0]]
-      },
-      {
+          "multiplier_shape": [[0]],
+          "fully_quantize": [False],
```

Comment:
Looks like we don't have good e2e test coverage for int8 cases. Could you make sure that this quantized op paths have enough e2e tests with the original TF models?

Reply:
Thank, I added some extra tests.

Reply:
Thanks for that.